sites/fast-website/src/app/components/side-navigation/side-navigation.ts

```diff
@@ -0,0 +1,48 @@
+import { attr, observable, FASTElement } from "@microsoft/fast-element";
+import {
+    CommunityContentPlacementData,
+    communityContentPlacementData,
+} from "../../data/community.data";
+import { throttle } from "lodash";
+
+export class SideNavigation extends FASTElement {
+    @attr
+    public category: string;
+
+    //can be removed to used sectionArray.length once index is restructured to have the correct amount of sections.
+    @attr
+    public sections: number;
+
+    @observable
+    public currentSection: number = 0;
+
+    @observable
+    public sectionArray: HTMLElement[] = Array.from(document.querySelectorAll("section"));
+
+    public updatecurrentSection = (): void => {
```

Comment:
```suggestion
    public updateCurrentSection = (): void => {
```

nit: camelCase

Reply:
Thank you for noticing this. I have updated it.